packages/manager/src/features/Databases/DatabaseCreate/DatabaseSummarySection.tsx

```diff
@@ -77,6 +74,15 @@ export const DatabaseSummarySection = (props: Props) => {
   ) : (
     'Please specify your cluster configuration'
   );
+  const getSuffix = (numberOfNodes: number) => {
+    let suffix = '';
+    if (numberOfNodes > 1) {
+      suffix = isNewDatabase ? 's - HA ' : 's: ';
+    } else {
+      suffix = isNewDatabase ? ' ' : ': ';
+    }
+    return suffix;
+  };
```

Comment:
It may be nice to extract this util function out of this component. Doing so would clean up this component a bit and even allow us to unit test `getSuffix` easily

Reply:
Also, can simplify by directly returning:

```suggestion
  const getSuffix = (numberOfNodes: number) => {
    if (numberOfNodes > 1) {
      return isNewDatabase ? 's - HA ' : 's: ';
    } else {
      return isNewDatabase ? ' ' : ': ';
    }
  };
```

Reply:
Thank you for the review! I've updated my PR